packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -242,7 +242,7 @@ function SendConfirmation(props: Props) {
         usdAmount,
         comment,
         recipient,
-        feeInfo,
+        feeEstimate?.feeInfo,
```

Comment:
I think we shouldn't move forward if `feeInfo` is undefined.
Otherwise we could risk sending a transaction where the actual fee hasn't been approved by the user.
or am I missing something?

Reply:
No, you're correct, see my other comment, fixed!